src/unstract/sdk/adapters/x2text/llm_whisperer_v2/src/static/json_schema.json

```diff
@@ -0,0 +1,156 @@
+{
+  "title": "LLM Whisperer X2Text v2",
+  "type": "object",
+  "required": [
+    "adapter_name",
+    "unstract_key",
+    "url"
+  ],
+  "properties": {
+    "adapter_name": {
+      "type": "string",
+      "title": "Name",
+      "default": "",
+      "description": "Provide a unique name for this adapter instance. Example: LLM Whisperer 1"
+    },
+    "url": {
+      "type": "string",
+      "title": "URL",
+      "format": "uri",
+      "default": "https://llmwhisperer-api.unstract.com",
+      "description": "Provide the URL of the LLM Whisperer service."
+    },
+    "unstract_key": {
+      "type": "string",
+      "title": "Unstract Key",
+      "format": "password",
+      "description": "API key obtained from the Unstract developer portal (https://unstract-api-resource.developer.azure-api.net)"
```

Comment:
Not through azure any more.  We have to get it from unified portal.
cc: @jagadeeswaran-zipstack @jaseemjaskp